.github/workflows/ci-actions.yml

```diff
@@ -0,0 +1,249 @@
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - "README.md"
+  pull_request:
+    types: [assigned, opened, synchronize, reopened, ready_for_review, edited]
+    paths-ignore:
+      - "README.md"
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+
+  build-artifacts:
+    name: Create web distribution
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone current windup-web
+        uses: actions/checkout@v2
+        with:
+          path: 'windup-web'
+      - name: Clone windup-web-distribution
+        uses: actions/checkout@v2
+        with:
+          repository: 'windup/windup-web-distribution'
+          path: 'windup-web-distribution'
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+
+      - uses: nanasess/setup-chromedriver@master
+      - name: maven-settings-xml-action
+        uses: whelk-io/maven-settings-xml-action@v13
+        with:
+          repositories: '[{"id":"sonatype-snapshots","url":"https://oss.sonatype.org/content/repositories/snapshots/"}, {"id":"jboss-releases", "url":"https://repository.jboss.org/nexus/content/repositories/thirdparty-releases/"}, {"id":"redhat-ga-repository", "url":"https://maven.repository.redhat.com/ga/"}, {"id":"central", "url":"https://repo.maven.apache.org/maven2"}]'
+          plugin_repositories: '[{"id":"redhat-ga-repository", "url":"https://maven.repository.redhat.com/ga/"}, {"id":"central", "url":"https://repo.maven.apache.org/maven2"}]'
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install Bower and PhantomJS
+        run: npm install -g bower phantomjs-prebuilt
+
+      - name: Build windup-web
+        run: mvn install -DskipTests=true -f windup-web
+      - name: Build windup-web-distribution
+        run: mvn install -DskipTests=true -f windup-web-distribution
+      - name: Rename distribution
+        run: mv windup-web-distribution/target/mta-web-distribution-*-with-authentication.zip mta-server.zip
```

Comment:
Why renaming? I think it would be better to have the full name with the version.

Reply:
It is necessary to rename it because I need the exact name of the file that will be uploaded to GH Actions, I mean this line https://github.com/carlosthe19916/windup-web/blob/861430e5e3d1351674ee11bc422c82712e1a3de7/.github/workflows/ci-actions.yml#L53

The name of the file to be uploaded must be static and written in the next step (see the link above)